pyaedt/modeler/Object3d.py

```diff
@@ -3832,6 +3833,28 @@ def __init__(self, component, *args, **kw):
         self._component = component
 
 
+class UserDefinedComponentProps(OrderedDict):
```

Comment:
@Samuelopez-ansys The description is missing here. If we need to merge before the milestone, we can do it later, no worries.